### PR TITLE
HOTFIX Add licence limitations

### PR DESCRIPTION
**Pull request by perrettecl**
Signed-off-by: Clement Perrette <clementperrette@eaton.com>

### CODE REVIEW

File: src/fty_alert_actions.cc [hunk 1]
```diff
@@ -845,6 +852,30 @@ s_handle_stream_deliver_asset(fty_alert_actions_t *self, fty_proto_t **asset_p,
     }
 }
 
+//  --------------------------------------------------------------------------
+//  Handle incoming metric through stream
+static void
+s_handle_stream_deliver_metric(fty_alert_actions_t *self, fty_proto_t *message)
+{
+    assert (self);
+    assert (msg_p);
```
Comment (jimklimov) [inline src/fty_alert_actions.cc]: `assert(message)`? Travis compiler fails here ;)

````
  CXX      src/src_libfty_alert_engine_la-fty_alert_actions.lo
src/fty_alert_actions.cc: In function 'void s_handle_stream_deliver_metric(fty_alert_actions_t*, fty_proto_t*)':
src/fty_alert_actions.cc:861:7: error: 'msg_p' was not declared in this scope
     assert (msg_p);
       ^
make[1]: *** [src/src_libfty_alert_engine_la-fty_alert_actions.lo] Error 1
````

File: src/fty_alert_actions.cc [hunk 2]
```diff
@@ -845,6 +852,30 @@ s_handle_stream_deliver_asset(fty_alert_actions_t *self, fty_proto_t **asset_p,
...
+
+        self->actionEnabled = (allowNotificationAndAction == 1);
+
+        if(self->actionEnabled)
+        {
+            log_debug("Actions and notification enabled");
+        }
+        else
+        {
+            log_debug("Actions and notification disabled");
```
Comment (jimklimov) [inline src/fty_alert_actions.cc]: Should this debug and one above clarify that this is about RC-0?

File: src/fty_alert_actions.cc [hunk 3]
```diff
@@ -845,6 +852,30 @@ s_handle_stream_deliver_asset(fty_alert_actions_t *self, fty_proto_t **asset_p,
...
+//  --------------------------------------------------------------------------
+//  Handle incoming metric through stream
+static void
+s_handle_stream_deliver_metric(fty_alert_actions_t *self, fty_proto_t *message)
+{
+    assert (self);
+    assert (message);
+    if (streq (fty_proto_name(message), "rackcontroller-0") && streq (fty_proto_type(message), "notification_and_action.global"))
+    {
+        int allowNotificationAndAction = std::stoi(fty_proto_value(message));
```
Comment (aquette) [inline src/fty_alert_actions.cc]: you should put this into a try/catch block, since it may throw an exception (either invalid or out of range)

**Comment by jimklimov** [comment]
So Jenkins run failed for reasons outside the codebase...

**Pull request merged**